app/util/analytics.py

```diff
@@ -194,8 +184,8 @@ def send_request(self):
                    "tool_ver": self.analytics.tool_version,
                    "exp_dur": self.analytics.duration,
                    "act_dur":  self.analytics.actual_duration,
-                   "sel_count": self.analytics.selenium_test_count,
-                   "jm_count": self.analytics.jmeter_test_count,
+                   "full_selenium": self.analytics.selenium_test_count,
+                   "full_jmeter": self.analytics.jmeter_test_count,
```

Comment:
suggest changing to `jmeter_test_rates` or `jmeter_test_rate_list` or `rate_by_jmeter_test_name`